packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1087,13 +1092,20 @@ class ThemeData with Diagnosticable {
   /// start using new colors, typography and other features of Material 3.
   /// If false, they will use the Material 2 look and feel.
   ///
+  /// If true, the default Typography will be [Typography.material2021],
+  /// otherwise it will default to [Typography.material2014].
+  /// 
   /// If a [ThemeData] is constructed with [useMaterial3] set to true,
   /// the default [typography] will be [Typography.material2021],
   /// otherwise it will be [Typography.material2014].
   ///
   /// However, just copying a [ThemeData] with [useMaterial3] set to true will
   /// not change the typography of the resulting ThemeData.
   ///
+  /// If true, the default [splashFactory] will be [InkSparkle] if and only if
+  /// the platform is Android and the app is compiled to run on the web.
```

Comment:
should this say "and the app is not running on the web"?

Reply:
Yep, fixed